bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties

```diff
@@ -1,7 +1,34 @@
 channel-type.system.signal-strength.label = Signalst�rke
+channel-type.system.signal-strength.state.option.0 = Kein Signal
+channel-type.system.signal-strength.state.option.1 = Schwach
+channel-type.system.signal-strength.state.option.2 = Durchschnittlich
+channel-type.system.signal-strength.state.option.3 = Gut
+channel-type.system.signal-strength.state.option.4 = Ausgezeichnet
 channel-type.system.low-battery.label = Niedriger Batteriestatus
 channel-type.system.battery-level.label = Batterieladung
 channel-type.system.trigger.label = Ausl�ser
 channel-type.system.rawbutton.label = Einfache Taste
 channel-type.system.button.label = Taste
 channel-type.system.rawrocker.label = Wipptaster
+channel-type.system.power.label = Power
+channel-type.system.location.label = Ort
+channel-type.system.location.description = Ort in geographischen Koordinaten (Latitude/Longitude/Altitude).
```

Comment:
(Latitude/Longitude/Altitude) is english. Please change to (Breitengrad/Längengrad/Höhe).

Reply:
Done. Thanks.